apps/web/src/server/db/schema.ts

```diff
@@ -117,6 +117,12 @@ export const verificationTokens = createTable(
   }),
 );
 
+export const roleEnum = pgEnum("role", ["admin", "user", "owner"]);
+
+export const roles = createTable("role", {
+  name: roleEnum("role").notNull().primaryKey(),
+});
+
```

Comment:
What's that?

Reply:
A new table for future implementation WIP